packages/advisor-components/src/RuleDetails/RuleDetails.tsx

```diff
@@ -0,0 +1,247 @@
+        </Stack>
+      </SplitItem>
+      <SplitItem>
+        <Stack>
+          {children && <StackItem>{children}</StackItem>}
+          <StackItem>
+            <Stack className="ins-c-rule-details__stack">
+              <StackItem>
+                <strong>{intl.formatMessage(messages.totalRisk)}</strong>
+              </StackItem>
+              <StackItem className="pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm">
+                <span className="ins-c-rule-details__stackitem">
+                  <span>
+                    <InsightsLabel value={rule.total_risk} rest={{}} />
+                  </span>
+                  <Stack hasGutter className="description-stack-override">
+                    <StackItem>
+                      <TextContent>
+                        <Text component={TextVariants.p}>
+                          {intl.formatMessage(messages.rulesDetailsTotalRiskBody, {
+                            risk: intl.formatMessage(messages[TOTAL_RISK_LABEL_KEY[rule.total_risk]] || messages.undefined).toLowerCase(),
+                            strong: (str) => <strong>{str}</strong>,
+                          })}
+                        </Text>
+                      </TextContent>
+                    </StackItem>
+                    <Stack>
+                      <StackItem>
+                        <SeverityLine
+                          className="severity-line"
+                          title={intl.formatMessage(messages.likelihoodLevel, {
+                            level: intl.formatMessage(messages[LIKELIHOOD_LABEL_KEY[rule.likelihood as Likelihood]]),
+                          })}
+                          value={rule.likelihood}
+                          tooltipMessage={intl.formatMessage(messages.likelihoodDescription, {
+                            level: intl.formatMessage(messages[LIKELIHOOD_LABEL_KEY[rule.likelihood as Likelihood]]).toLowerCase(),
+                          })}
+                        />
+                      </StackItem>
+                      <StackItem>{renderImpactSeverityLine()}</StackItem>
+                    </Stack>
+                  </Stack>
+                </span>
+              </StackItem>
+              {resolutionRisk && resolutionRiskDesc && (
+                <React.Fragment>
+                  <hr></hr>
+                  <StackItem>
+                    <strong>{intl.formatMessage(messages.riskOfChange)}</strong>
+                  </StackItem>
+                  <StackItem className={`pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm`}>
+                    <span className="ins-c-rule-details__stackitem">
+                      <span>
+                        <InsightsLabel
+                          text={intl.formatMessage(messages[RISK_OF_CHANGE_LABEL_KEY[resolutionRisk as RiskOfChange]])}
+                          value={resolutionRisk}
+                          hideIcon
+                          rest={{}}
```

Comment:
Does passing empty object do anything?

Reply:
Mhh, might be a good to pass default value for the `rest` so we don't have to pass empty objects.

Reply:
Checked the source code, the `rest` doesn't have a default value. The component just spreads it. If empty object,nothing is spread:
```
...
    <Label {...rest} {...props} className={classNames(className, `ins-c-label-${value}`)} icon={!hideIcon && VALUE_TO_STATE[value].icon}>
      {text || VALUE_TO_STATE[value].text}
    </Label>
```

Reply:
Yeah, that's the issue. I'll open a PR to fix that (let's keep the empty onject here for backwards compatibility)